src/index.html

```diff
@@ -22,6 +22,402 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="/"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="https://apple.com"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://samsung.com"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://sv.wikipedia.org/wiki/Smartmobil"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://en.wikipedia.org/wiki/Laptop"
+              data-qa="nav-hover"
```

Comment:
The 'data-qa' attribute is used for testing purposes. Ensure that its use here is intentional and that it follows the testing strategy of the project.